src/main/java/com/epam/preprod/sirenko/entity/Cart.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.preprod.sirenko.entity;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Cart {
```

Comment:
Why do we need it?We already have cart in cartDao

Reply:
I guess I'm still trying to use DB. Deleted this class